src/App.js

```diff
@@ -17,19 +17,30 @@ export default class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
+      authed: false,
       recommendationData: undefined,
       foodSearchTerm: '',
       showFilter: false,
       dietFilter: '',
       totalCount: 0,
     }
     this.toggleFilterMenu = this.toggleFilterMenu.bind(this)
+    this.onLoginClick = this.onLoginClick.bind(this)
```

Comment:
If you set the method within the class as a () => {} you shouldn't have to use bind here.

Reply:
👍 I should use a code mode to update all components.